benchmark/buffers/buffer-compare-offset.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+var common = require('../common.js');
+
+var bench = common.createBenchmark(main, {
+  method: ['offset', 'slice'],
+  size: [16, 512, 1024, 4096, 16386],
+  millions: [1]
+});
+
+function compareUsingSlice(b0, b1, size, iter) {
+  const len = size >> 2;
+  for (var i = 0; i < iter; i++)
+    Buffer.compare(b0.slice(1, len), b1.slice(1, len));
+}
+
+function compareUsingOffset(b0, b1, size, iter) {
+  const len = size >> 2;
+  for (var i = 0; i < iter; i++)
+    b0.compare(b1, 1, 1 + len, 1, 1 + len);
+}
+
+function main(conf) {
+  const iter = (conf.millions >>> 0) * 1e6;
+  const size = (conf.size >>> 0);
+  const b0 = Buffer.alloc(size, 'a');
+  const b1 = Buffer.alloc(size, 'b');
+
+  b1[size - 1] = 'b'.charCodeAt(0);
+
+  switch (conf.method) {
+    case 'slice':
+      bench.start();
```

Comment:
What about moving the `bench.start()`/`bench.end()` to be closer to the actual loop in the respective functions?